src/components/Heading/Heading.tsx

```diff
@@ -49,14 +51,15 @@ export const Heading = chakra(
       id,
       level = HeadingLevels.Two,
       size,
+      noSpace,
       text,
       url,
       urlClass,
       ...rest
     } = props;
     const finalLevel = getMappedLevel(level);
     const variant = size ? getVariant(size, HeadingSizes) : `h${finalLevel}`;
-    const styles = useStyleConfig("Heading", { variant });
+    const styles = useStyleConfig("Heading", { variant, noSpace });
```

Comment:
This is not needed because it's not being used in the relevant `src/theme/components/heading.ts` file. This is okay to remove.

Reply:
Update: let's keep this here and instead remove line 95 `mb={noSpace ? "0" : null}`.